backend/de.metas.contracts/src/main/java/de/metas/contracts/pricing/SubscriptionPricingRule.java

```diff
@@ -119,9 +119,13 @@ private IPricingResult aggregateSubscriptionDiscount(
 	{
 		if (subscriptionDiscountLine != null
 				&& !subscriptionPricingResult.isDisallowDiscount()
-				&& (subscriptionDiscountLine.isPrioritiseOwnDiscount() || !subscriptionPricingResult.isDiscountCalculated()) )
+				&& (subscriptionDiscountLine.isPrioritiseOwnDiscount() || !subscriptionPricingResult.isDiscountCalculated()))
 		{
 			subscriptionPricingResult.setDiscount(subscriptionDiscountLine.getDiscount());
+			if (subscriptionDiscountLine.isPrioritiseOwnDiscount())
+			{
+				subscriptionPricingResult.setDiscountEditable(false);
```

Comment:
ok, so i guess that's the answer to my question above. 
but as i wrote, i think `IsDiscountEditable` is related to the user, not to following rules.
Therefore i think we need an additional property like `dontOverrideDiscountAdvice` or something

Reply:
added `isDontOverrideDiscountAdvice`